# #异常捕获
# try:
#     a = 1
#     print(a)
# except BaseException as e:# 捕获异常并重命名打印
#     print(e)
# else:
#     print("没有异常执行这里！")
# finally:
#     print("有没有异常都执行这里！")





